code/modules/mob/living/carbon/species.dm

```diff
@@ -1175,7 +1172,9 @@ var/list/has_died_as_golem = list()
 
 	gender = NEUTER
 
-	tox_mod = 2
+	tox_mod = 9.9
+    burn_mod = 9.9
+    brute_mod = 9.9
```

Comment:
I think these have bad indent

Reply:
nah man certified legit game balance

Reply:
Remember to use Tab for indenting (making the space beneath the code), not space bar